src/Console/BaseCommand.php

```diff
@@ -178,8 +190,12 @@ public function run(array $argv, ConsoleIo $io): ?int
             $io->setInteractive(false);
         }
 
-        /** @var int|null */
-        return $this->execute($args, $io);
+        $this->dispatchEvent('Command.started', ['args' => $args]);
```

Comment:
Not sure about the naming, the event is *before* the command execution starts.

Reply:
My thinking was, that the actuall command execution is the main part of which users may want to get an event triggered - not the argument parsing. 
But I agree that the naming can be more specific 😁